apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -474,9 +532,15 @@ const (
 	// Details: https://kubernetes.io/docs/reference/using-api/server-side-apply
 	ApplyStrategyTypeServerSideApply ApplyStrategyType = "ServerSideApply"
 
-	// ApplyStrategyTypeReportDiff will generate a report of the difference between
-	// the resource on the member cluster and to be placed resource snapshot from the hub periodically.
-	ApplyStrategyTypeReportDiff ApplyStrategyType = "ReportDiff"
+	// ApplyStrategyTypeReportApplyDiff will use server-side apply to apply manifests and report
+	// that a drift has occurred if fields managed by Fleet have been overwritten by agents
+	// other than Fleet.
+	ApplyStrategyTypeReportApplyDiff ApplyStrategyType = "ReportApplyDiff"
+
+	// ApplyStrategyTypeReportAnyDiff will use server-side apply to apply manifests and report
+	// that a drift has occurred if any field has been added, removed, or edited by agents
+	// other than Fleet, even if the field is not under Fleet's management.
+	ApplyStrategyTypeReportAnyDiff ApplyStrategyType = "ReportAnyDiff"
```

Comment:
what happens if user wants to "applyIfNoDiff"?

Reply:
Updated 👌

Reply:
Oh, this one was a leftover; will remove.